sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/InMemoryCatalog.scala

```diff
@@ -73,6 +79,8 @@ class InMemoryCatalog extends ExternalCatalog {
     }
   }
 
+  private val fs = FileSystem.get(new Configuration)
```

Comment:
The passed in database location URI is already qualified, so I think it's safe to use a empty configuration to get the file system. Please correct me if I was wrong. cc @yhuai